blog/test_ndb.py

```diff
@@ -5,20 +5,7 @@
 from google.appengine.ext import testbed
 
 from bz_blog.wsgi import application
-
-
-class TestBlog(ndb.Model):
-    title = ndb.StringProperty(required=True)
-    slug = ndb.StringProperty(required=True)
-    body = ndb.BlobProperty(required=True)
-    posted = ndb.DateTimeProperty(auto_now_add=True)
-    author = ndb.StringProperty()
-    blob_key = ndb.BlobKeyProperty(required=False)
-    thumnail = ndb.ComputedProperty(lambda self: images.get_serving_url(self.blob_key, size=32) if self.blob_key else '')
-
-    @classmethod
-    def query_blog(cls, ancestor_key):
-        return cls.query(ancestor=ancestor_key).order(-cls.posted)
+from blog.models import Blog
 
 
 class DatastoreTestCase(unittest.TestCase):
```

Comment:
No need to init self.testbed as it is iniitalled in unittest.TestCase. 

Use  super(DatastoreTestCase, self).setUp()

Also no need to call self.testbed.deactivate() as it will be called from base class.